### PR TITLE
Draft: Search API Endpoint

### PR DESCRIPTION
**Pull request by Gurder29**
I added Search-contacts.php in the LAMP folder, I just require a bit of assistance to test it

### CODE REVIEW

File: LAMP/SearchContacts.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	$searchCount = 0;
+
+	$conn = new mysqli("localhost", "TheBeast", "WeLoveCOP4331", "COP4331");
+	if ($conn->connect_error)
+	{
+		returnWithError( $conn->connect_error );
+	}
+	else
+	{
+		$stmt = $conn->prepare("select FirstName from Contacts where FirstName like ? and UserID=?");
```
Comment (kingcolston) [inline LAMP/SearchContacts.php]:
```suggestion
		$stmt = $conn->prepare("SELECT * FROM Contacts WHERE FirstName LIKE ? AND UserID=?");

```

Comment (kingcolston) [reply LAMP/SearchContacts.php]: * gives us all the information from the table regarding any relation to that person which is probably what we need for the details panel. I also capitalized the key selector words just for good styling practices.

File: LAMP/SearchContacts.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+		$FirstName = "%" . $inData["search"] . "%";
+		$stmt->bind_param("ss", $colorName, $inData["userId"]);
```
Comment (kingcolston) [inline LAMP/SearchContacts.php]: colorName should be FirstName

**Review by kingcolston — COMMENTED** [review]
Going to hold off and not merge this in just yet but so far so good. Ideally Line 17 should be changed to `$row` since we want the array of all the searches in this case.

**Review by cgrady3 — APPROVED** [review]
I'd like to get together with you and compare our work, mostly so I can understand what's going on

**Pull request merged**